### PR TITLE
feat: adds cozy.client.files.createDirectoryRecursively()

### PR DESCRIPTION
**Pull request by gregorylegarec**
Very useful for creating konnectors folder.

### CODE REVIEW

**Comment by gregorylegarec** [comment]
@Cpatchane I totally agree but I lack the time to write tests. Maybe I should add a TODO aboit that ?

File: docs/README.md [hunk 1]
```diff
@@ -352,6 +352,19 @@ const created = await cozy.client.files.createDirectory({
 ```
 
 
+### `cozy.client.files.createDirectoryRecursively(path)`
+
+`cozy.client.files.createDirectory(path)` is used to create one or more directories for a given path.
```
Comment (enguerran) [inline docs/README.md]: misnamed `createDirectory` instead of `createDirectoryRecursively`

**Comment by enguerran** [comment]
If you miss time to cover code with tests, you may miss the point of *clean code*.

I don't think it is a good excuse.

I think a `TODO` is something that will never be done.

*My 2 cents with software craftsmanship inside*

**Comment by gregorylegarec** [comment]
@enguerra I totally agree with you. I am just priorizing _clean sleep_ and _clean health_ over _clean code_.

**Comment by gregorylegarec** [comment]
Ok I see no simple way to test this method easily, at last with the current mocking system based on MockAPI.

I tried to use sinon spies and stubs, without success.

**Pull request closed without merging**